frontends/precinct-scanner/src/screens/scan_warning_screen.tsx

```diff
@@ -37,6 +38,20 @@ import { AppContext } from '../contexts/app_context';
 import { toSentence } from '../utils/to_sentence';
 import { useSound } from '../hooks/use_sound';
 
+const ResponsiveButtonParagraph = styled.p`
+  @media (orientation: portrait) {
```

Comment:
Any reason we can't apply these styles to both portrait and landscape? Or alternatively, could we tweak the styles here such that they work for both portrait and landscape?

Reply:
We can apply this styles to both orientations, however if we apply these styles to the screen the buttons will become as wide as the widest text block on the page:
<img width="1319" alt="image" src="https://user-images.githubusercontent.com/28444/197895593-29603458-357d-4c8b-a843-e7a40699b088.png">

These styles were added to "tweak" the styles so that the button layout looked good on portrait as they already looked good on landscape.

What is the problem that you're hoping to solve?

Reply:
Got it!

Not at all a problem to have media queries that target specific orientations! In this case, the desired layouts in portrait vs. landscape just struck me as the same, so it seemed possible to accomplish what we wanted with the same styles in all orientations. Good to know about the widening challenge specific to landscape, though, which I now see was the reason that we needed to apply these styles behind a media query.